pkg/authorization/conditions/path.go

```diff
@@ -0,0 +1,32 @@
+package conditions
+
+import (
+	"fmt"
+	"net/http"
+	"regexp"
+
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/apis/sessions"
+)
+
+type path struct {
+	path *regexp.Regexp
```

Comment:
Maybe call this `regexp` or something? Else it's `path.path` which seems weird to me?